plenum/config.py

```diff
@@ -26,10 +26,20 @@
 
 baseDir = '~/.plenum/'
 walletsDir = 'wallets'
+NODE_BASE_DATA_DIR = baseDir
 nodeDataDir = 'data/nodes'
 clientDataDir = 'data/clients'
+LOG_DIR = os.path.join(baseDir, "log")
+GENERAL_CONFIG_DIR = '/etc/indy'
```

Comment:
Do we need this value here? Can we remove it?

Reply:
wait for Sergey's fix